releases/v0.9.0-alpha0.md

```diff
@@ -0,0 +1,76 @@
+# Release Notes for v0.9.0-alpha
+
+Release Date: April 25th, 2024
+
+This is our first release based on Kata main, but it is an alpha release that supports
+only a subset of features.
+
+This release is based on [3.4.0](https://github.com/kata-containers/kata-containers/releases/tag/3.4.0) of Kata
+and [v0.9.0](https://github.com/confidential-containers/enclave-cc/releases/tag/v0.9.0) of enclave-cc.
+
+This release supports pulling images inside of the guest with some caveats but it does
+not support pulling encrypted or signed images inside the guest.
+This release supports attestation and includes the guest components in the rootfs,
+but it does not support any TEE platform.
+Peer pods is also not supported.
+
+This release was created mainly for development purposes. For a full feature set,
+consider returning to v0.8.0 or using the next release.
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
```

Comment:
enclave-cc gained several improvements (I don't mean the dependabot PRs ;) but in one sentence

* enclave-cc moved to a unified LibOS bundle for secure rootfs key handling and to the latest Occlum v0.30.1 release that adds SGX EDMM support for dynamically adjusting the enclave size.